src/main/java/gyro/aws/apigatewayv2/DeploymentFinder.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2020, Brightspot.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.aws.apigatewayv2;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import gyro.aws.AwsFinder;
+import gyro.core.Type;
+import software.amazon.awssdk.services.apigatewayv2.ApiGatewayV2Client;
+import software.amazon.awssdk.services.apigatewayv2.model.Api;
+import software.amazon.awssdk.services.apigatewayv2.model.Deployment;
+
+/**
+ * Query Deployment.
+ *
+ * Example
+ * -------
+ *
+ * .. code-block:: gyro
+ *
+ *    deployment: $(external-query aws::deployment {api-id: "tf2x92hetc"})
+ */
+@Type("deployment")
```

Comment:
Similar as above to make it less generic
```suggestion
@Type("api-gateway-deployment")
```